jenkins/scripts/fetch_logs.sh

```diff
@@ -46,31 +47,66 @@ else
   fi
 fi
 
-# Send Remote script to Executer
-scp \
-  -o StrictHostKeyChecking=no \
-  -o UserKnownHostsFile=/dev/null \
-  -i "${METAL3_CI_USER_KEY}" \
-  "${CI_DIR}/files/run_fetch_logs.sh" \
-  "${METAL3_CI_USER}@${TEST_EXECUTER_IP}:/tmp/" > /dev/null
 
-echo "Fetching logs"
-# Execute remote script
-# shellcheck disable=SC2029
-ssh \
-  -o StrictHostKeyChecking=no \
-  -o UserKnownHostsFile=/dev/null \
-  -o ServerAliveInterval=15 \
-  -i "${METAL3_CI_USER_KEY}" \
-  "${METAL3_CI_USER}"@"${TEST_EXECUTER_IP}" \
-  PATH=/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/bin \
-  /tmp/run_fetch_logs.sh "logs-${BUILD_TAG}.tgz" \
-  "logs-${BUILD_TAG}" "${IMAGE_OS}" "${TESTS_FOR}"
+if [ "${BARE_METAL_LAB}" == true ]; then
+  # Send Remote script to BML server
+  scp \
+    -o StrictHostKeyChecking=no \
```

Comment:
This is excessively repetitive and error-prone way to call ssh/scp.

I'd suggest defining the common options something like this:

```bash
declare -a SSH_OPTIONS=(
    -o StrictHostKeyChecking=no
    -o UserKnownHostsFile=/dev/null
    -o ServerAliveInterval=15
    -i "${METAL3_CI_USER_KEY}"
    -o ProxyCommand="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${METAL3_CI_USER_KEY} -W %h:%p ${METAL3_CI_USER}@${TEST_EXECUTER_IP}"
)
```
and then all the calls would looks like
```bash
ssh \
  "${SSH_OPTIONS[@]}" \
  <user>@<port> \
  <whatever command>
```

Reply:
Thank you for pointing it out.
Changed it according your suggestion.